.github/tests/key_test.py

```diff
@@ -1,55 +1,46 @@
-import json
-from configparser import ConfigParser
-
-filename_eng = "en/live/global.ini"
-filename_ger = "live/global.ini"
-
-
-def addLine(filename, line):
-    with open(filename, encoding="utf_8_sig", mode="r+") as file:
-        file_data = file.read()
-        file.seek(0, 0)
-        file.write(line + ("\n") + file_data)
-
-
-def removeFirstLine(filename):
-    with open(filename, encoding="utf_8_sig", mode="r+") as file:
-        lines = file.readlines()
-        file.seek(0, 0)
-        file.truncate()
-        file.writelines(lines[1:])
-
-
-line = "[DEFAULT]"
-file_data = ""
-
-addLine(filename=filename_eng, line=line)
-addLine(filename=filename_ger, line=line)
-
-config_eng = ConfigParser(
-    allow_no_value=True, comment_prefixes=None, delimiters=("="), interpolation=None
-)
-config_ger = ConfigParser(
-    allow_no_value=True, comment_prefixes=None, delimiters=("="), interpolation=None
-)
-config_eng.read(filename_eng, "utf_8_sig")
-config_eng_section = config_eng["DEFAULT"]
-
-config_ger.read(filename_ger, "utf_8_sig")
-config_ger_section = config_ger["DEFAULT"]
-
-not_found_keys = {}
-
-line = 1
-for key in config_eng_section.keys():
-    value = config_ger_section.get(key)
-    if value == None:
-        not_found_keys[line] = key
-    line += 1
-
-removeFirstLine(filename_eng)
-removeFirstLine(filename_ger)
-
-if len(not_found_keys):
-    print(json.dumps(not_found_keys, indent=4))
-    exit(1)
+import configparser
+
+
+def keys_in_second_ini(first_file, second_file):
+    """
+    :param first_file: The path to the first ini file.
+    :param second_file: The path to the second ini file.
+    :return: True if all keys in the first ini file are present in the second ini file, False otherwise.
+    """
+    # Parse the two ini files
+    first_ini = configparser.ConfigParser()
```

Comment:
Hier würde ich mindestens noch comment_prefixes=None übergeben, da wir keine Kommentare in der ini haben.
Es gibt viele Zeilen mit ; und # die nicht für Kommentare stehen. (Auch wenn es hier uns nur um die Keys geht)

Reply:
Ebenso wie allow_no_value=True